File: portfolio/utils/logger.ts
class Logger {
	constructor() {}

	static success(message: string, schemaType: string) {
		console.log(` ✅  ${schemaType}: ${message}`)
	}

	static error(message: string, schemaType: string) {
		console.log(` ❌  ${schemaType}: ${message}`)
	}

	static info(message: string, schemaType: string) {
		console.log(` ℹ️  ${schemaType}: ${message}`)
	}

	static warn(message: string, schemaType: string) {
		console.log(` ⚠️   ${schemaType}: ${message}`)
	}
}

export { Logger }

File: tests/utils/logger.py
class Logger:
    @staticmethod
    def success(msg):
        print("\n ✅  {}".format(msg))

    @staticmethod
    def error(msg):
        print("\n ℹ️  {}".format(msg))

    @staticmethod
    def warning(msg):
        print("\n ⚠️   {}".format(msg))

    @staticmethod
    def error(msg):
        print("\n 🛑  {}".format(msg))

